### ISSUE TITLE
Learning rate and momentum?

### ISSUE THREAD

**Issue by log69** [tangledpath/ruby-fann#21]
Hi, is there an option anywhere to specify the learning rate and / or the momentum for the training? Thanks.

**Comment by git-steven** [comment]
You can call 
```ruby
fann.set_learning_momentum(momentum) 
# and
fann.set_learning_rate(rate)
